### ISSUE TITLE
Jinja comments in HTML language

### ISSUE THREAD

**Issue by OmeGak** [samuelcolvin/jinjahtml-vscode#75]
It seems like Better Jinja is hijacking the comment style of the `html` language.

How to reproduce:
1. Open a new file and set the language to `html`.
1. Write some lines of HTML code.
1. Toggle line comment.
1. Note that the inserted comment is `{# #}` instead of `<!-- -->`.

Disabling this extension fixes the issue.

**Comment by samuelcolvin** [comment]
This is the intended behaviour for most people I think.

I have no idea if it could be made configurable. If you can find a way, I'd be hapy to accept a pr.

**Comment by OmeGak** [comment]
I'm curious. What is the rationale for this being the intended behavior? 

When I work on a HTML Jinja template I set the language to `jinja-html`. When I work on a HTML file, I set the language to `html`. I would expect Jinja comments only in the `jinja-html` language, not in `html` too. Since the extension is contributing all these Jinja-specific languages, I don't really understand what the purpose of inserting Jinja syntax in non-Jinja languages.

The way it is now forces me to disable the extension every time I need to work on HTML files, which seems backward to me. I use VScode for multiple projects. Some of them using HTML Jinja templates, some of them not. One of them uses both HTML Jinja templates and plain HTML files. Going nuts here.
